sormas-backend/src/main/java/de/symeda/sormas/backend/visit/VisitFacadeEjb.java

```diff
@@ -623,7 +627,11 @@ private void onVisitChanged(VisitDto existingVisit, Visit newVisit) {
 		}
 
 		if (newVisit.getCaze() != null) {
-			caseService.updateFollowUpUntilAndStatus(newVisit.getCaze());
```

Comment:
What's the reason for removing this line of code?

Reply:
`caseFacade.saveCase()` will call `updateFollowUpUntilAndStatus`

Reply:
Makes sense!